### ISSUE TITLE
Current Technologies

### ISSUE THREAD

**Issue by christophernhill** [USRSE/monthly-community-calls#1]
**Jordan Perr-Sauer**: A more technology focused discussion where practitioners of research software engineering can share their favorite new technologies and advocate for best practices.

Some ongoing work in this space with HPC-BP [https://csmd.ornl.gov/highlight/hpc-best-practices-webinar-series](https://csmd.ornl.gov/highlight/hpc-best-practices-webinar-series)

**Comment by christophernhill** [comment]
@bernhold - great thanks, good to hear from you. I was migrating our USRSE community call proposed topics list from a clunky spreadsheet to issues. 

Hopefully we can have a community call sometime soon on this topic.

**Comment by vsoch** [comment]
Yes! I think this discussion would also shed light on the diversity of things that are worked on. My current sources of "What is new and cool" tend to be Twitter, Reddit, and Hacker News :)

**Comment by mtwest2718** [comment]
It's important to decide whether this is a formal presentation situation or more a "look at this cool thing."

**Comment by vsoch** [comment]
I think "look at this cool thing" would be ok? Don't a lot of "listen to my professional talk on `$thing`" start out as "Look at this cool `$thing`!"

**Comment by mtwest2718** [comment]
I like the rather informal "look at this `cool_thing`" option. It lowers the threshold for folks not normally comfortable with public speaking and it lessens the stress of having to make a formal presentation.

Also there are lots of places to give SOP formal talks online right now. Duplicating that environment feels a bit much.

**Comment by jdamerow** [comment]
Yes, we had something like a short lightning talk style presentation of a tool or best practice with questions in breakout rooms after. Not like a "professional talk",  just to share something cool!

**Comment by sk8forether** [comment]
> @bernhold - great thanks, good to hear from you. I was migrating our USRSE community call proposed topics list from a clunky spreadsheet to issues.

@christophernhill This seems like a good candidate for a [GitHub project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards)!  Assuming you want the added organization/automation.
